R/runNMDS.R

```diff
@@ -56,20 +28,14 @@
 #' @param ... additional arguments to pass to \code{FUN} and
 #'   \code{nmds.fun}.
 #'
-#' @param dimred String or integer scalar specifying the existing dimensionality
+#' @param dimred \code{Character scalar} or \code{integer scalar}. Specifies the existing dimensionality
 #'   reduction results to use.
 #'
-#' @param ndimred Integer scalar or vector specifying the dimensions to use if
+#' @param ndimred \code{Integer scalar} or \code{integer vector}. Specifies the dimensions to use if
```

Comment:
This could be just \code{Integer vector} since scalar is vector with one element